gef-ui/server/controllers/activities-controller/index.test.js

```diff
@@ -34,6 +64,38 @@ describe('mapPortalActivities', () => {
       text: '',
       datetime: { timestamp: fromUnixTime(1638458265), type: 'datetime' },
       byline: { text: 'Bob Smith' },
+      html: '',
+      facilityType: '',
+      facilityID: '',
+      maker: '',
+      checker: '',
+    }];
+
+    expect(response).toEqual(expected);
+  });
+
+  it('should return formatted array for mojTimeline for facility submission', () => {
+    const response = mapPortalActivities(facilityActivity);
+
+    // expected format
+    const expected = [{
+      label: { text: 'Bank facility stage changed' },
+      text: '',
+      datetime: { timestamp: fromUnixTime(1638458265), type: 'datetime' },
+      byline: { text: 'Bob Smith' },
+      html: 'facility',
+      facilityType: 'Cash facility',
```

Comment:
isn't this supposed to be `Cash`/`Contingent` etc?

Reply:
that is for making a string, for the link so needs to have it as that